iml-agent/src/action_plugins/stratagem/action_purge.rs

```diff
@@ -54,7 +54,7 @@ pub async fn read_mailbox(
         .try_filter_map(future::ok)
         .chunks(rmfids_size)
         .map(|xs| xs.into_iter().collect())
-        .try_for_each_concurrent(10, |fids| {
+        .try_for_each(|fids| {
```

Comment:
Why?

Reply:
Should have been in action_filesync.rs, and I didn't back it out